### ISSUE TITLE
Add a 'Skip to Main' button at the top to improve UX for keyboard users.

### ISSUE THREAD

**Issue by satnaing** [satnaing/shadcn-admin#22]
This one should be beginner-friendly. So, I'll mark this as `good first issue`.
I might request changes on the PR or co-author if necessary for this.

![skip-to-main](https://github.com/user-attachments/assets/83f9aa42-5d51-4020-8b96-55d9db7fb4d2)

**Comment by ahnge** [comment]
![Screenshot from 2024-08-25 13-09-46](https://github.com/user-attachments/assets/1438924d-cdb9-4753-9ef4-59ac08384a1b)
Like this? What about on mobile?
![Screenshot from 2024-08-25 13-15-09](https://github.com/user-attachments/assets/7e060655-4162-4ff0-8789-9f19ac26e485)
And what should that button do?

**Comment by satnaing** [comment]
@ahnge 
No, you can go the [astro-paper](https://astro-paper.pages.dev/) or [my website](https://satnaing.dev/) and see that in action.
If you go to above websites and press `tab` in your keyboard, the button will appear. You can forget about the mobile.


https://github.com/user-attachments/assets/af2d7631-c3c2-4448-9dae-70bb2e4ba929

**Comment by ahnge** [comment]
Understood. But, what would the main content be? To where do I scoll on click?

**Comment by thantko20** [comment]
would love to work on this if this is still open!

**Comment by satnaing** [comment]
To `<main id="content" ...` 

> Understood. But, what would the main content be? To where do I scoll on click?

**Comment by ahnge** [comment]
Got it! Let me work on this one. Thanks.

**Comment by satnaing** [comment]
> would love to work on this if this is still open!

Thanks. But for now, I'll assign this to @ahnge since he comes here first.
However, you can contribute to other issues if you want. @thantko20
